heron/examples/src/java/com/twitter/heron/examples/ExclamationTopology.java

```diff
@@ -14,8 +14,10 @@
 
 package com.twitter.heron.examples;
 
+import java.util.List;
 import java.util.Map;
 
+import com.twitter.heron.api.topology.IUpdatable;
```

Comment:
Should we use heron raw api directly @kramasamy 

Even if we try to use heron api in examples, should we add a new example rather than reuse this ExclamationTopology?